2-run-automation-test.md

```diff
@@ -0,0 +1,131 @@
+# Running Automation test with Kobiton
+## 1. Prepare Kobiton configuration for excuting automation testing
+### 1.1 Get Kobiton Username and API Key
++ Go to  https://portal.kobiton.com
+
+**a. Get Kobiton Username**
+  + In the upper right hand corner, click on User icon and in the drop down menu, select 'Profile' 
+
+![alt text](./assets/click_profile.png )
+
++ You should see your username
+
+![alt text](./assets/username.png )
+
+**b.Get Kobiton API key**
++ Click on user icon in the upper right hand corner again and select 'Setting'
+
+![alt text](./assets/click_setting.png )
+
++ You will see your API key under the 'API Keys' 
+
+![alt text](./assets/api_key.png )
+
+1.2 Configure in your project TeamCity
++ In your project in TeamCity, on the left side on the page, click on 'Build Step' to set build steps
++ On ther right side on the page, Choose 'Command Line ' on the combo box 'Runner type'
```

Comment:
"On ther right side on the page, Choose 'Command Line ' on the combo box 'Runner type'" ->
"On the right side of the page, for the 'Runner type' category, choose 'Command Line'. 

Make sure you only capitalize words at the beginning of sentences or if it's a proper noun :)